### PR TITLE
clientv3: threadsafe

### PR DESCRIPTION
**Pull request by xiang90**
/cc @gyuho @heyitsanthony @mitake

### CODE REVIEW

File: clientv3/kv.go [hunk 1]
```diff
@@ -60,10 +62,11 @@ type KV interface {
 }
 
 type kv struct {
+	c *Client
```
Comment (gyuho) [inline clientv3/kv.go]: Is there any special reason that this got moved to top?

Comment (xiang90) [inline clientv3/kv.go]: yes. the mu does not guard c since it is immutable.

Comment (gyuho) [inline clientv3/kv.go]: Makes sense. Yeah I see it a few lines below. Thanks.

**Comment by gyuho** [comment]
LGTM.

**Pull request merged**